### ISSUE TITLE
Can it be looked up in the local site by downloading the whole schemecookbook.org.tgz

### ISSUE THREAD

**Issue by APIPLM** [schemedoc/cookbook#14]
As in my site,  it is needed a proxy to access the host `schemecookbook.org`. That means I can not see this cookbook at all. But the cookbook look likes so amazing to have.
So I downloaded the file schemecookbook.org.tgz from [in you site](https://misc.lassi.io/2019/schemecookbook.org.tgz). Upzip to it the files in my laptop. Then go the folder `..\schemecookbook.org\Cookbook` , edit the file` index.html `in Emacs. 

1.  Find the tag `<div contentbox>` , and put corsal position there.
2.  `set-mark_command`,  `Forward Tag` in html-mode. 
3.  copy-paste the content to  the temp buffer. 
4.  see the temp buffer by the command `browse-url-of-buffer`

And I got the item list of the cookbook  in the webpage. Then by the item name in this page I can find which one is what I am interesting,  then come to the corresponding folder in the unzipped folder `schemecookbook.org ` for the webpage(`index.html`) of each item. Of course edit ` index.html ` in Emacs as the same way I did in the above.

For the search ability,  like `grep `the title or  code in the target folder  `schemecookbook.org`.
The question is that by the above way to reference the cookbook we have out there. Is_ it missing a lots?

**Comment by lassik** [comment]
`schemecookbook.org.tgz` is a copy of the old Schematics Scheme Cookbook. It has been downloaded automatically from web.archive.org, and most of the contents are probably out of date for current Scheme. We intended to start with the Schematics cookbook, and add new content to make it up-to-date. But the Schematics cookbook is using a license that seems difficult, so we decided to start the new cookbook from nothing.

@jcubic is gathering the new cookbook in [the issues](https://github.com/schemedoc/cookbook/issues). If you have questions about how to do something in Scheme, please open issues and we can answer them. Then we can collect the answers into web pages.

**Comment by APIPLM** [comment]
Yes. It is a pretty old one as I came back to check the unzipped folder. Once I have an issue in Scheme, I open it here.
